project/src/test/java/com/google/sps/EventSpannerTasksTest.java

```diff
@@ -205,11 +209,29 @@ public void testEventRegistrationDoGet() throws Exception {
     Assert.assertEquals(returnedEvent.getName(), EVENT_NAME);
     Assert.assertEquals(returnedEvent.getDescription(), DESCRIPTION);
     Assert.assertEquals(returnedEvent.getLocation(), LOCATION);
-    Assert.assertEquals(returnedEvent.getDate(), DATE);
+    Assert.assertEquals(returnedEvent.getDate(), DATE_FUTURE);
     Assert.assertEquals(returnedEvent.getTime(), TIME);
     Assert.assertTrue(
         new ReflectionEquals(returnedEvent.getAttendees(),/*excludeFields= */ null)
             .matches(new HashSet<User>(Arrays.asList(ATTENDEE, HOST))));
   }
-}
 
+  /**
+   * Verify that events in the past are NOT retrieved
+   */
+  @Test
+  public void testGetFutureEventsOnly() throws Exception {
```

Comment:
you can include the result in the test name as opposed to writing a comment.

Reply:
That makes sense, done